pkgs/development/libraries/libctl/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, guile
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libctl";
+  version = "4.5.1";
+
+  src = fetchFromGitHub {
+    owner = "NanoComp";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "uOydBWYPXSBUi+4MM6FNx6B5l2to7Ny9Uc1MMTV9bGA=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook guile pkg-config gfortran ];
+  configureFlags = [ "--enable-shared" ];
+  meta = with lib; {
```

```suggestion

  meta = with lib; {
```
(the blank line is not necessary but increases readability of the code)

---

I made the suggested change. Does it look good now?